manifests/stage-nv-peer-mem-driver/0020_role.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: gpudirect-driver
+  namespace: {{ .RuntimeSpec.Namespace }}
```

Comment:
the namespace here will be set to mlnx-network-operator-resources however i did not see the operator has permissions to create Role in this namespace

Reply:
Missed & added.